### DISCUSSION TITLE
What highlights are you using to get the look shown on the README's gif

### DISCUSSION THREAD

**Discussion by daniel-hernandez-sony** [mrjones2014/legendary.nvim#422] [General]
Thank you for making this plugin, it's super helpful.

I want to ask if anyone knows how did the author managed to get the look of the UI shown in the gif. I love how it looks, with an opaque background and the prompt title with a stark green  when calling the `:Legendary` command.

I really wanna have a similar look for my telescope / vim.ui.select but I don't really know where to start looking!

**Comment by mrjones2014** [comment]
Hey there, author here 👋 

I'm using @olimorris's _awesome_ [onedarkpro.nvim](https://github.com/olimorris/onedarkpro.nvim) theme, and you can see my configuration for it [in my dotfiles repo here](https://github.com/mrjones2014/dotfiles/blob/14b2be035561421a82519d9622a6c950a619be95/nvim/lua/my/configure/theme.lua). There's also some other cool configurations in [this pinned issue](https://github.com/olimorris/onedarkpro.nvim/issues/31) of the `onedarkpro.nvim` colorscheme plugin.

**Reply by mrjones2014** [reply]
In my config I'm setting some custom Telescope highlights to get it looking like that.

**Reply by daniel-hernandez-sony** [reply]
Thank you so much! And I doubly appreciate that my question was not really related to the plugin itself. Looking forward to your future stuff!
